pkg/modules/resiliency.go

```diff
@@ -37,6 +37,11 @@ var (
 	XCSIPodmonEnabled = "X_CSI_PODMON_ENABLED"
 )
 
+const (
+	ControllerMode = "controller"
+	NodeMode       = "node"
```

Comment:
It seems these constants are utilized within this package. Please check if we can make them private.